README.md

```diff
@@ -25,7 +25,7 @@ You can find the complete ILLIXR system [here][38].
 
 ILLIXR also provides its components in standalone configurations to enable architects and
     system designers to research each component in isolation.
-The standalone components are packaged together in the [v1-latest release][39] of ILLIXR. 
+The standalone components are packaged together in the [v3.1.0 release][39] of ILLIXR. 
```

Comment:
Let's say something like "as of v3.x.x" or something, so we don't have to update this every version update.